pandas/io/common.py

```diff
@@ -345,6 +346,10 @@ def get_handle(
         Mode to open path_or_buf with.
     encoding : str or None
         Encoding to use.
+    errors : str, default 'strict'
```

Comment:
I think we called this encoding_errors elsewhere (as errors is too generic here). cc @WillAyd @TomAugspurger

Reply:
The discussion is at https://github.com/pandas-dev/pandas/pull/27899. I don't recall the resolution.

Reply:
I pushed for passing that to a handle prior to the call in that conversation and just documenting that. But I think this is OK as well

Reply:
This is identical to the keyword argument names of the standard Python `open()` function. In my opinion this allows us to point users to the standard python library documentation for more information on what these arguments mean.